v-lazy-image/index.js

```diff
@@ -69,7 +71,7 @@ export default {
     });
 
     onBeforeUnmount(() => {
-      if ("IntersectionObserver" in window && state.observer) {
+      if (state.observer) {
```

Comment:
I'm not sure if this condition is needed - if it's not met in `onMounted` hook, observer will be null.
Also - no reactive state is dependent on `state.observer`, so it can be as well in a simple variable, instead of a reactive object.